backend/de.metas.purchasecandidate.base/src/main/java/de/metas/purchasecandidate/purchaseordercreation/localorder/PurchaseOrderFromItemFactory.java

```diff
@@ -105,6 +106,8 @@ private PurchaseOrderFromItemFactory(
 
 	public void addCandidate(final PurchaseOrderItem purchaseOrderItem)
 	{
+		orderFactory.externalPurchaseOrderUrl(purchaseOrderItem.getExternalPurchaseOrderUrl());
```

Comment:
`ExternalPurchaseOrderUrl` needs to be part of the `PurchaseOrderAggregationKey`